contributing/community/review-comments.rst

```diff
@@ -0,0 +1,177 @@
+Respectful Review Comments
+==========================
+
+:doc:`Reviewing issues and pull requests </contributing/community/reviews>`
+is a great way to get started with contributing to the Symfony community.
+Anyone can do it! But before you give a comment, take a step back and think,
+is what you are about to say actually what you intend?
+
+Communicating over the Internet with nothing but text can pose a
+big challenge, especially if you remember that the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+ideas and opinions.
+
+Not everyone speaks English or is able to use a keyboard. Some might
+have Dyslexia or similar conditions that affect their writing.
+
+Not to mention that some might have a bad experience from previous
+contributions (to other projects).
+
+You're not alone in this. This guide will try to help you write
+constructive, respectful and helpful reviews and replies.
+
+.. tip::
+
+    This guide is not about lecturing you to "conform" or give-up
+    your ideas and opinions but helping you to better communicate,
+    prevent possible confusion, and keeping the Symfony community a
+    welcoming place for everyone. **You are free to disagree with
+    someone's opinions, just don't be disrespectful.**
+
+First of, accept that many programming decisions are opinions.
+Discuss trade offs, which you prefer, and reach a resolution quickly.
+It's not about being right or wrong, but using what works.
+
+Tone of voice
+-------------
+
+We don't expect you to be completely formal, or to even write error-free
+English. Just remember one thing: Don't swear, and be respectful to others.
```

Comment:
"Just one thing" and then list 2?   😉

Reply:
You can just drop "`n` thing(s)"

Reply:
First thins first, but not necessarily in that order - Doctor Who 😄